sw/device/lib/testing/randomness_quality.h

```diff
@@ -0,0 +1,52 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#ifndef OPENTITAN_SW_DEVICE_LIB_TESTING_RANDOMNESS_QUALITY_H_
+#define OPENTITAN_SW_DEVICE_LIB_TESTING_RANDOMNESS_QUALITY_H_
+
+#include <stdint.h>
+
+#include "sw/device/lib/base/status.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif  // __cplusplus
+
+/**
+ * Test failure probability setting.
+ *
+ * The statistical tests in this file will always fail with some probability,
+ * even on truly random data. Use these settings to toggle the threshold
+ * values.
+ */
+typedef enum randomness_quality_threshold {
+  /**
+   * Set thresholds for a 5% failure rate on truly random data (alpha = 0.05).
+   */
+  kRandomnessQualityThresholdFivePercent,
+  /**
+   * Set thresholds for a 1% failure rate on truly random data (alpha = 0.01).
+   */
+  kRandomnessQualityThresholdOnePercent,
+} randomness_quality_threshold_t;
```

Comment:
nit: Would it make sense to rename these to something like `SignificanceLevelFivePercent` or `AlphaFivePercent` because they are not really numerical thresholds, but sort of "selectors" of the thresholds?

Reply:
I like the "significance level" one, that seems to be the right statistical term here. I'll call it `kRandomnessQualitySignificanceFivePercent` to match the codebase convention of prefixing constants with the file name (and removing the `Level` just to make that long name sightly less long).